code/game/objects/explosion.dm

```diff
@@ -138,3 +150,74 @@ proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impa
 proc/secondaryexplosion(turf/epicenter, range)
 	for(var/turf/tile in trange(range, epicenter))
 		tile.ex_act(2)
+
+
+/client/proc/check_bomb_impacts()
+	set name = "Check Bomb Impact"
+	set category = "Debug"
+
+	var/newmode = alert("Use reactionary explosions?","Check Bomb Impact", "Yes", "No")
+
+	var/turf/epicenter = get_turf(src)

```

Comment:
Can be resolved by making src usr instead (I know usr is bad but it's a debug verb I think we can get away with it)